.github/workflows/docker-build-on-commit.yaml

```diff
@@ -148,6 +167,8 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           push: ${{ !startsWith(github.ref, 'refs/heads/dependabot/') }}
+          build-args: |
+            BASE_IMAGE_TAG=25-alpine-${{ needs.prepare-variables.outputs.tag_build_number }}
```

Comment:
Please put to BASE_IMAGE_TAH full tag name like: `ghcr.io/netcracker/qubership-java-base:25-alpine-${{ needs.prepare-variables.outputs.tag_build_number }}`